firmware/controllers/sensors/redundant_sensor.cpp

```diff
@@ -9,33 +9,66 @@ RedundantSensor::RedundantSensor(SensorType outputType, SensorType first, Sensor
 {
 }
 
-void RedundantSensor::configure(float maxDifference, bool ignoreSecondSensor) {
+void RedundantSensor::configure(float maxDifference, bool ignoreSecondSensor, float secondaryMaximum) {
 	m_maxDifference = maxDifference;
 	m_ignoreSecond = ignoreSecondSensor;
+	m_partialSecondMaximum = secondaryMaximum;
 }
 
 SensorResult RedundantSensor::get() const {
-	auto result1 = Sensor::get(m_first);
+	// Sensor 1 is always full range, i.e. 0% -> 100%
+	auto sensor1 = Sensor::get(m_first);
+
+	// If sensor 1 result is invalid, return inconsistency error
+	if (!sensor1.Valid) {
+		return UnexpectedCode::Inconsistent;
+	}
 
 	// If we're set to disable redundancy, just pass thru the first sensor
 	if (m_ignoreSecond) {
-		return result1;
+		return sensor1;
 	}
 
-	auto result2 = Sensor::get(m_second);
+	// Sensor 2 may be partial, i.e. 0% -> m_partialSecondMaximum%
+	auto sensor2 = Sensor::get(m_second);
 
-	// If either result is invalid, return invalid.
-	if (!result1.Valid || !result2.Valid) {
+	// If sensor 2 result is invalid, return inconsistency error
+	if (!sensor2.Valid) {
 		return UnexpectedCode::Inconsistent;
 	}
 
-	// If both are valid, check that they're near one another
-	float delta = absF(result1.Value - result2.Value);
-	if (delta > m_maxDifference) {
-		return UnexpectedCode::Inconsistent;
+	if (m_partialSecondMaximum == 0) {
```

Comment:
This gets changed in the second PR to default to `100`, vs "special case `0`" as here.